### DISCUSSION TITLE
Can you please add these channe;s

### DISCUSSION THREAD

**Discussion by rogedelos** [iptv-org/community#1140] [Q&A]
Chanel 7 Melbourne, Chanel 7 Brisbane, Chanel 7 Sydney, Chanel seven perth, Channel 7 act, channel 7 mate Vic, Channel 7 mate sydney, chanel 7 mate perth, chanel 7 mate brisbane, chanel 7 mate  Fox sports 501, fox sports 502, fox sports 503, fox sports 504, fox sports 505, fox sports 506, fox sports 507, fox sports UHD 1, fox sports UHD 2, fox sports uhd 3, fox sports uh4 please add these thank you

**Comment by freearhey** [comment]
If the request is still relevant, please repost it using this [form](https://github.com/iptv-org/iptv/issues/new?assignees=&labels=channel+search&projects=&template=4_channel-search.yml&title=Find%3A+).
